### ISSUE TITLE
Load CSV file

### ISSUE THREAD

**Issue by bersace** [orellazri/renderkit#66]
Hi,

Thanks for renderkit, it look awesome. I was wondering for such project for a few monthes ! I need to load CSV file, what do you think of it ?

Regards,
Étienne

**Comment by orellazri** [comment]
Hey! Thanks, that’s great to hear!
Do you mean using a CSV file as a datasource? If so, how would the key-value pairs be laid out? Perhaps in a way that each line will contain `key,value`?

Let me know if that’s not what you meant :)

**Comment by bersace** [comment]
Yep, I mean CSV datasource. The simplest case is just a two dimensional array of strings.

``` 
{{ range .Csv }}
{{   range .Columns }}
{{     . }}
{{   end }}
{{ end }}
```

Would it work like that ?

**Comment by orellazri** [comment]
@bersace we currently support key-value pairs, that are merged from all datasources, so in your case the template will still reference the variables from the key names like with the other datasources.
I'm not sure how useful it would be then, but this can be considered.
You can also feel free to create a PR if you'd like!
